src/miniasync.def

```diff
@@ -13,3 +13,4 @@ EXPORTS
     vdm_delete
     vdm_descriptor_threads
     vdm_descriptor_threads_polled
+    os_rand_r
```

Comment:
1. I approve vs/gha changes but we cannot merge this change in .def file.
2. We decided to use #ifdef platform.h instead of force include.

Reply:
I fixed those errors on my PR (#41 ) and added changes from this PR to it.